roles/common/templates/etc/serverspec/os_spec.rb

```diff
@@ -0,0 +1,153 @@
+require 'spec_helper'
+
+describe command('cat /etc/pam.d/login|grep @include') do
+  cmd_stdout=['@include common-auth',
+          '@include common-account',
+          '@include common-session',
+          '@include common-password']
+  cmd_stdout.each do |line|
+    its(:stdout) {should contain line }
+  end
+end
+
+describe command('cat /etc/pam.d/common-password |grep success=7') do
+  cmd_stdout="password [success=7 default=ignore] pam_unix.so obscure sha512\n"
+  its(:stdout) {should eq cmd_stdout}
+end
+
+describe file('/etc/login.defs') do
+  it { should contain 'PASS_MAX_DAYS   90' }
+  it { should contain 'PASS_MIN_DAYS   1' }
+end
+
+describe file('/etc/ssh/sshd_config') do
+  it {should contain 'PermitEmptyPasswords no'}
+  it {should contain 'PubkeyAuthentication yes'}
+  it {should contain 'PasswordAuthentication no'}
+  it {should contain 'PermitRootLogin without-password'}
+  it {should contain 'RSAAuthentication yes'}
+  it {should contain 'HostbasedAuthentication no'}
+  it {should contain 'IgnoreRhosts yes'}
+  it {should contain 'PrintMotd yes'}
+  it {should contain 'PermitUserEnvironment no'}
+  it {should contain 'StrictModes yes'}
+  it {should contain 'ServerKeyBits 1024'}
+  it {should contain 'TCPKeepAlive yes'}
+  it {should contain 'LoginGraceTime 120'}
+  it {should contain 'MaxStartups 100'}
+  it {should contain 'LogLevel INFO'}
+  it {should contain 'MaxAuthTries 5'}
+  it {should contain 'Protocol 2'}
+  it {should contain 'GatewayPorts no'}
+  it {should contain 'UsePAM yes'}
```

Comment:
Might want to have these start with a `^` to prevent caching commented out lines that would mach here.